packages/pegasus/src/pegasus.js

```diff
@@ -20,11 +19,11 @@ import { makeCourierMaker, getCourierPK } from './courier.js';
 const DEFAULT_DENOM_TRANSFORMER = IBCSourceTraceDenomTransformer;
 const DEFAULT_TRANSFER_PROTOCOL = ICS20TransferProtocol;
 
-const TRANSFER_PROPOSAL_SHAPE = {
+const TransferProposalShape = M.splitRecord({
   give: {
-    Transfer: null,
+    Transfer: AmountShape, // TODO get amount shape from brand
```

Comment:
I like `makeNatAmountShape(brand)` for now.

Reply:
Hi @Chris-Hibbert (cc @michaelfig since it's pegasus)
TIL `makeNatAmountShape`. Nice!

But from a brief skim of the code, I did not figure out what expression to write in this location for the brand. ELI5, could you show me plausible code? Thanks.

Reply:
`denomState.localBrand` seems plausible.

Given the uncertainty, it seems worthwhile to add tests:
 - offers that use the right brand work
 - offers that use the wrong brand don't work